insights/core/exceptions.py

```diff
@@ -81,10 +84,20 @@ def __init__(self, content_type):
 
 class SkipException(SkipComponent):
     """
+    .. warning::
+        This class is deprecated, please use
+        :py:class:`insights.core.exceptions.SkipComponent` instead.
+
     Exception that should be thrown from parsers that are explicitly
     written to look for errors in input data.  If the expected error
     is not found then the parser should throw this exception to
     signal to the infrastructure that the parser's output should not be
     retained.
     """
-    pass
+    def __init__(self):
+        deprecated(
+            SkipComponent,
```

Comment:
Oops, I missed this one, here is should be `SkipException`

Reply:
Ahh good catch sorry about that, fixed.

Reply:
NEVER mind, it always happens :)